models/event_filters/stripe_charges.sql

```diff
@@ -5,7 +5,7 @@ with charges as (
 )
 
 select
-  id,
+  data__object__id as id,
```

Comment:
Ok -- good call. I actually implemented the same pattern across all of the other event types (first make an event stream model and then dedup into a model that has single record for each object PK). You can see this in `subscription_events` and `subscriptions`. I renamed this file to `stripe_charge_events.sql` to be consistent.

My $.02 is that you should continue to use this data source instead of the `stripe_charges` table in the integration--I have found a large number of small annoying cases in the data where building the models on the higher-fidelity event stream allows me to make choices that are much better than what exists in those other tables. If you check out `invoices`, for example, there is some logic in there that fixes erroneous values in the `forgiven` field.